Readme change

thejohnjansen opened the pull request:
Adding a list of chapters to help track tests as they either migrate from old-tests, or if we create new tests. Putting this in 'old-tests' because that seems like a good spot to start when creating tests.

thejohnjansen committed:
Add list of chapters to the readme

We discussed having a way to track tests for WebDriver. I figured a good
starting point would be to simply list the chapters and mark them as
"not ready" for tests, or "have coverage".

thejohnjansen committed:
meant to mark Actions as 'red'

sideshowbarker commented:
Shouldn’t this change be made to the https://github.com/w3c/web-platform-tests/blob/master/webdriver/README.md file instead? Otherwise, why we do still have webdriver tests in the `oldtests` directory? Is there a plan to move them to the top-level `webdriver` directory eventually?

andreastt commented:
The intention is to use the tests in _/old-tests_ for inspiration. This change should go in _/webdriver/README.md_ as you say.

thejohnjansen committed:
changing location of the list of chapters

Request was made to put the list inside /webdriver/ instead of
/old-tests/webdriver/

thejohnjansen committed:
Bad merge

Fixing the merge to keep the old description and add the new instuctions
and chapter list

thejohnjansen commented:
Updated to put this change into the webdriver directory instead of the old-tests directory.

andreastt commented on webdriver/README.md:
```diff
@@ -1,8 +1,132 @@
+so that different driver implementations can determine 
+whether they meet the recognised standard. 
+
+## How to run the tests
```
This whole section is wrong.

thejohnjansen replied:
I didn't write that part, it was already there. Do you want me to remove it for this PR?

andreastt commented on webdriver/README.md:
```diff
@@ -1,8 +1,132 @@
+
+Note: that you will likely need to start the driver's server before running.
+
+## Updating configuration
```
This should also be removed.

thejohnjansen replied:
Fixed

andreastt commented on webdriver/README.md:
```diff
@@ -1,8 +1,132 @@
+
+Be sure not to commit your _webdriver.cfg_ changes when your create or modify tests.
+
+## How to write tests
```
This is also outdated.

thejohnjansen replied:
Fixed

andreastt commented on old-tests/webdriver/README.md:
```diff
@@ -12,12 +12,12 @@ whether they meet the recognized standard.
    `WD_BROWSER=chrome python runtests.py`
 
-To be run a specific test file you can just run `python test_file.py`
+To run a specific test file you can just run `python test_file.py`
```
Please revert changes to the file in old-tests.

thejohnjansen replied:
Done, but for some reason the final line still shows as different. I ran the 'rich diff' and it said there was nothing to show. I'm not sure how to fix that.

thejohnjansen replied:
SWEET! Fixed. Thanks for that.

andreastt commented:
Please address code review comments and squash.

thejohnjansen committed:
Address code review feedback

thejohnjansen committed:
remove line break

thejohnjansen committed:
fixing lint errors

andreastt commented on old-tests/webdriver/README.md:
```diff
@@ -35,4 +35,4 @@ Be sure not to commit your _webdriver.cfg_ changes when your create or modify te
    should be stored in a folder named 'res'.
 3. Test name should explain the intention of the test e.g. `def
-   test_navigate_and_return_title(self):`
+   test_navigate_and_return_title(self):`
```
This should not change

thejohnjansen replied:
Fixed

thejohnjansen committed:
see if this worked...
